modules/ROOT/pages/getting-started.adoc

```diff
@@ -11,61 +11,73 @@ NOTE: For more information on configuration, refer to the documentation for xref
 
 To launch FCOS on AWS:
 
-. Go to the https://getfedora.org/coreos/download/[download page] to identify the correct AMI.
+. Identify the correct AMI image ID for your region from https://getfedora.org/coreos/download/[the download page]
 
-. Specify the Ignition configuration file as the https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-add-user-data[user-data]. You can skip this step, but you will want to specify a key using `--key-name` so that you can still access the node. This provides an easy way to test out FCOS without first creating an Ignition configuration.
+. Launch the VM using `aws ec2 run-instances`. The Ignition file can be passed to the VM as its https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-add-user-data[user data]. You can skip this step, but you will want to specify a key using `--key-name` so that you can still access the node. This provides an easy way to test out FCOS without first creating an Ignition configuration.
 
-.Example launching Fedora CoreOS on AWS using an Ignition configuration file
+.Example launching FCOS on AWS using an Ignition configuration file
 [source, bash]
 ----
 aws ec2 run-instances <other options> --image-id <ami> --user-data file://example.ign
 ----
 
-.Example launching Fedora CoreOS on AWS using a registered SSH key pair
+.Example launching FCOS on AWS using a registered SSH key pair
 [source, bash]
 ----
 aws ec2 run-instances <other options> --image-id <ami> --key-name my-key
 ----
 
 While the AWS documentation mentions cloud-init and scripts, FCOS does not support cloud-init or the ability to run scripts from user-data. It accepts only Ignition configuration files.
 
-=== Launching with QEMU
-. Go to the https://getfedora.org/coreos/download/[download page] to retrieve the latest image suitable for QEMU.
+=== Launching with QEMU or libvirt
+. https://getfedora.org/coreos/download/[Download and verify] the latest image suitable for QEMU.
 
 . Uncompress the file. Assuming that the downloaded file is called `fedora-coreos-qemu.qcow2.xz`, the command would be the following:
 +
-`unxz fedora-coreos-qemu.qcow2.xz`
-+
-. Ensure the correct ownership of the file:
-+
-`chown core:kvm fedora-coreos-qemu.qcow2`
+[source, bash]
+----
+unxz fedora-coreos-qemu.qcow2.xz
+----
 +
-TIP: The above example assumes that you have created the user named `core` on your local machine. If you did not, replace `core` with the user that will boot FCOS on the local host.
+TIP: Make sure that your user has access to `/dev/kvm` either by doing `chmod 666 /dev/kvm` (this matches modern distros), or if `/dev/kvm` is owned by the `kvm` group, adding yourself to that group with `usermod -aG kvm $USER`.
```

Comment:
Nit: I'd drop the `chmod` and just keep the reference to the device and the group.